### PR TITLE
Added support for a specific directory in demo command

### LINKED ISSUE #154: Add a directory argument to the demo command
Hi guys!

Small detail - I was just using the new `symfony demo` command (yay!) and I immediately wrote `symfony demo foo_bar_baz` - to make sure it didn't run over any existing directories. But, this argument doesn't exist.

I think it makes sense to have this argument (but not the version argument) - it's consistent with `symfony new`.

Thanks!

### PR DESCRIPTION
**Pull request by lyrixx**
fixes #154

### CODE REVIEW

File: src/Symfony/Installer/DemoCommand.php [hunk 1]
```diff
@@ -38,22 +38,29 @@ protected function initialize(InputInterface $input, OutputInterface $output)
     {
         parent::initialize($input, $output);
 
-        $this->projectDir = getcwd();
+        if ('symfony_demo' === $input->getArgument('directory')) {
```
Comment (xabbuh) [inline src/Symfony/Installer/DemoCommand.php]: I would do this only if the argument has not been given at all, but not depending on the vaue if given.

Comment (lyrixx) [inline src/Symfony/Installer/DemoCommand.php]: indeed; fixed.

**Comment by javiereguiluz** [comment]
Thanks @lyrixx.

File: src/Symfony/Installer/DownloadCommand.php [hunk 2]
```diff
@@ -150,6 +152,19 @@ protected function download()
         return $this;
     }
 
+    protected function checkProjectName()
```
Comment (stof) [inline src/Symfony/Installer/DownloadCommand.php]: you should remove the method from NewCommand too

Comment (lyrixx) [inline src/Symfony/Installer/DownloadCommand.php]: https://github.com/symfony/symfony-installer/pull/157

**Pull request merged**